birdy/client/base.py

```diff
@@ -165,11 +165,33 @@ def _method_factory(self, pid):
 
         process = self._processes[pid]
 
-        input_defaults = OrderedDict()
-        for inpt in process.dataInputs:
-            iid = sanitize(inpt.identifier)
-            default = getattr(inpt, "defaultValue", None) if inpt.dataType != 'ComplexData' else None
-            input_defaults[iid] = utils.from_owslib(default, inpt.dataType)
+        def sort_inputs_key(i):
+            """Sort inputs for function creation.
```

Comment:
Maybe a comment: "function used as key in sort ..."

Reply:
I changed the docstring